easybuild/easyconfigs/f/FIX/FIX-1.068-foss-2019a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'FIX'
+version = '1.068'
+
+homepage = 'https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX'
+description = """FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad
+components can be removed from the 4D FMRI data."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://www.fmrib.ox.ac.uk/~steve/ftp/']
+sources = ['fix.tar.gz']
+checksums = ['bb10a24f28b3c26a382744faa127c1c7bf3167dc7a4f06da1aab89b58b74ac72']
+
+dependencies = [
+    ('FSL', '6.0.2', '-Python-3.7.2'),
```

Comment:
change to: `('FSL', '6.0.2', versionsuffix),`

Reply:
Done.